hedyorg/hedy
# 💻 Language switcher text

Fixes #4835 

[Babel](https://www.unicode.org/cldr/charts/44/supplemental/territory_language_information.html) only lists language codes in Latin alphabet. I did some searching and I think it would be quite difficult to find the non Latin language codes. So for the non Latin languages, I thought the best solution would be to just state the _non Latin language_ in the switcher instead of the _non Latin language code._ 

**How to test**
1. Select a non latin language in your profile
2. Go to Hedy
3. Check if there is a language in the switcher (and it's not None)
4. Switch between English and the (non Latin) language

https://github.com/hedyorg/hedy/assets/48122190/73adcb1c-fefb-42f3-958d-3876dbf6eb7b
